boomerang.js

```diff
@@ -1520,7 +1524,9 @@ BOOMR_check_doc_domain();
 				xhr = new (BOOMR.orig_XMLHttpRequest || BOOMR.window.XMLHttpRequest)();
 				xhr.open("POST", impl.beacon_url);
 				xhr.setRequestHeader("Content-type", "application/x-www-form-urlencoded");
-				xhr.send(paramsJoined);
+				if( impl.beacon_auth_token !== "")
```

Comment:
add braces around the if block.  make sure your code passes our eslint checks before committing.